.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+src/js/*.config.js
```

Comment:
How sure are we that we don't want to share config files for stuff? I could see this blowing up on our faces in the future.